### ISSUE TITLE
Class 'Mailgun\Mailgun' not found

### ISSUE THREAD

**Issue by anupalhub** [ajfranzoia/cakephp-mailgun#2]
I have cloned the plugin in app/Plugin/Mailgun 
Also loaded the plugin with CakePlugin::load('Mailgun'); in bootstrap. 
I also added the configuration you told in email class in app/Config

But when i try to send the email it gives error Class 'Mailgun\Mailgun' not found
Any help please?

**Comment by anupalhub** [comment]
As per the issue closed #1 
I removed the line <code>use Mailgun\Mailgun </code>
Now i get the error: Class 'Mailgun' not found 

I have manually installed the plugin. Is anything else i need to do ?

**Comment by ajfranzoia** [comment]
Usage of Composer for installing is strongly recommended. Otherwise, if you have manually copied the file, besides removing that line you will have to include Mailgun class in your code (using php `require` function).

**Comment by anupalhub** [comment]
The below is the structure after copying manually. 

![mailgun tree](https://cloud.githubusercontent.com/assets/7344503/6863018/3f0f2796-d470-11e4-96e4-090d2631a006.PNG)
I have also added the <code>require('Mailgun\Mailgun.php');</code> instead of <code>use 'Mailgun/Mailgun'</code>

But still i get missing <code>class 'Mailgun' not found</code> error in <code>app\Plugin\Mailgun\Lib\Network\Email\MailgunTransport.php</code>

Any thing else i need to do?

**Comment by jetsetting-stressball** [comment]
I had a similar problem.  The root issue was that although my composer file had the Mailgun/Mailgun-php reference in it, I did NOT have the dependencies it needs.  See this page for more details:

https://github.com/mailgun/mailgun-php

The three additional references I had to add to my composer.json file, plus Mailgun, are:

```
"Mailgun/Mailgun-php": "^1.0",
"php-http/curl-client": "^1.0",
"guzzlehttp/psr7": "^1.0",
"php-http/message": "^1.0",
```

Hope that helps people avoid the issues I did.

-j
